scripts/integrations/main.py

```diff
@@ -0,0 +1,89 @@
+import typer
+import configparser
+import json
+import os
+import sys
+import shutil
+from rich import print
+
+app = typer.Typer()
+base_integration_path = "../integrations"
```

Comment:
In general I think it's more reliable to create a subdirectory than to mess with files outside the script's local directory. If we want to support relative paths like this, it may be a better choice to let the user choose the value of this, with a default value like `./integrations`.

Reply:
Agree